pandas/core/tools/datetimes.py

```diff
@@ -773,8 +773,14 @@ def to_datetime(
 
     format : str, default None
         The strftime to parse time, e.g. :const:`"%d/%m/%Y"`. Note that
-        :const:`"%f"` will parse all the way up to nanoseconds. See
-        `strftime documentation
+        :const:`"%S"` without :const:`"%f"` will capture fractional 
+        seconds up to nanoseconds, Note that :const:`"%f"` will parse
+        up to nanoseconds. 
+        :const:`"%S"` without :const:`"%f"` will capture up to
+        nanoseconds if present as decimal places, but 
+        will also handle the case where the number of seconds is an
```

Comment:
can you add an example which illustrates this case (or both)

Reply:
If u don't mind, Can u help me with how I can add an example?

Reply:
You need to add another section in the same docstring. See this document: https://numpydoc.readthedocs.io/en/latest/format.html#examples

Note that the order of sections is important. You can check other docstrings in the pandas docs for samples, and to check which data we usually use...

Also, the CI is red because of black. You can run black before committing, and check the contributing docs for further information: https://pandas.pydata.org/pandas-docs/stable/development/contributing.html